tests/src/CoreMangLib/system/delegate/VSD/VSD_ResolveWorker.cs

```diff
@@ -0,0 +1,55 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Please open an issue for the test gap for VM side of delegate testing in coreclr repo